projects/com.oracle.truffle.llvm.parser.base/src/com/oracle/truffle/llvm/parser/base/model/enums/Linkage.java

```diff
@@ -46,7 +46,8 @@
     AVAILABLE_EXTERNALLY,
     LINKERPRIVATE,
     LINKERPRIVATE_WEAK,
-    LINKONCEODRAUTOHIDE;
+    LINKONCEODRAUTOHIDE,
+    UNKNOWN; // TODO: required by LLVM IR Parser, should be removed when no longer needed
```

Comment:
I think it is not a problem, as long as we remove it again when we remove the text parser.